src/cli/commands/run.js

```diff
@@ -30,8 +30,17 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
     if (!visitedBinFolders.has(binFolder)) {
       if (await fs.exists(binFolder)) {
         for (const name of await fs.readdir(binFolder)) {
-          binCommands.push(name);
-          scripts[name] = quoteForShell(path.join(binFolder, name));
+          //consider only .cmd scripts on Windows and show them without the extension - #624
+          if (process.platform === 'win32') {
+            if (name.indexOf('.cmd') !== -1) {
```

Comment:
Might be more "correct" to make sure that the name ends with ".cmd". This just checks that it is anywhere in the string (so "test.cmd.bak" would match).

Maybe use `name.match(/\.cmd$/)` instead?